tfexec/import.go

```diff
@@ -75,8 +67,8 @@ func (opt *VarFileOption) configureImport(conf *importConfig) {
 	conf.varFile = opt.path
 }
 
-func (t *Terraform) Import(ctx context.Context, opts ...ImportOption) error {
-	importCmd := t.ImportCmd(ctx, opts...)
+func (t *Terraform) Import(ctx context.Context, address, id string, opts ...ImportOption) error {
```

Comment:
I have these as explicit strings here instead of the options model since they are required for this command.

Reply:
This is reasonable, but it means a breaking change for tfexec if `terraform import` gains or loses required arguments during future `terraform` versions.

To my knowledge there is no other `terraform` subcommand with required arguments, but I may be wrong...